.travis.yml

```diff
@@ -19,12 +19,13 @@ before_install:
   - curl --version
   - wget --version
 install:
-  - if [ -z "${SHELLCHECK-}" ]; then nvm install node && npm install && npm prune && npm ls urchin doctoc eclint; fi
+  - if [ -z "${SHELLCHECK-}" ]; then nvm install node && npm install && npm prune && npm ls urchin doctoc eclint dockerfile_lint; fi
   - '[ -z "$WITHOUT_CURL" ] || sudo apt-get remove curl -y'
 script:
   - if [ -n "${MAKE_RELEASE-}" ]; then export GIT_EDITOR="sed -i '1 s/^/99.99.99 make release test/'" && git fetch --unshallow --tags && echo proceed | make TAG=99.99.99 release ; fi
   - if [ -n "${DOCTOCCHECK-}" ]; then cp README.md README.md.orig && npm run doctoc && diff -q README.md README.md.orig ; fi
   - if [ -n "${ECLINT-}" ]; then npm run eclint ; fi
+  - if [ -n "${DOCKERFILE_LINT-}" ]; then npx dockerfile_lint ; fi
```

Comment:
npx should not be used in scripts; if we want to depend on something, it should be a dependency. since this is a dependency, we should set up an npm run-script.

Reply:
Okay, updated.